petastorm/spark/__init__.py

```diff
@@ -0,0 +1,15 @@
+#  Copyright (c) 2020 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .spark_dataset_converter import make_spark_converter  # noqa: F401
```

Comment:
We also need to expose SparkDataConverter and its public methods.

Reply:
Curious, what's the use-case for that?